src/character.cpp

```diff
@@ -655,7 +655,7 @@ int Character::effective_dispersion( int dispersion ) const
     /** @EFFECT_PER penalizes sight dispersion when low. */
     dispersion += ranged_per_mod();
 
-    dispersion += encumb( body_part_eyes ) / 2;
+    dispersion *= vision_score();
```

Comment:
The first difficulty here is that dispersion should increase, not decrease with a low vision score. The second difficulty is that dispersion modifiers should be additive, not multiplicative, from what Kevin said on discord.
```suggestion
    float vision = vision_score();
    if( vision == 0.0f ) { // replace with <= something once know what should be
        dispersion += 10000;
    } else {
        dispersion += std::min( 10000, std::round( ( 57.0f / vision ) - 57.0f ) );
    }
```
This may need formatting, and the 57.0f is an initial estimate until we can get some data on effective dispersion (not sure why this isn't showing up on the test results in terms of the reported dispersion....).